diff --git a/node_modules/yargs-interactive/src/yargs-interactive.js b/node_modules/yargs-interactive/src/yargs-interactive.js
index 62f0a45..f08a656 100755
--- a/node_modules/yargs-interactive/src/yargs-interactive.js
+++ b/node_modules/yargs-interactive/src/yargs-interactive.js
@@ -3,8 +3,11 @@ const interactiveMode = require('./interactive-mode');
 const filterObject = require('./filter-object');
 const isEmpty = require('./is-empty');
 
+const argsProvided = process.argv.slice(2);
+const isArgProvided = (arg) => argsProvided.some(argProvided => argProvided === `--${arg}` || argProvided.startsWith(`--${arg}=`));
+
 // Set up yargs options
-let yargsInteractive = (processArgs = process.argv.slice(2), cwd) => {
+let yargsInteractive = (processArgs = argsProvided, cwd) => {
   const yargsConfig = yargs(processArgs, cwd);
 
   // Add interactive functionality
@@ -39,6 +42,10 @@ let yargsInteractive = (processArgs = process.argv.slice(2), cwd) => {
         return true;
       }
 
+      if (item.prompt === 'if-no-arg') {
+        return !isArgProvided(key);
+      }
+
       // Cases: item.prompt === "if-empty" or item.prompt undefined (fallbacks to "if-empty")
       // Prompt the items that are empty (i.e. a value was not sent via parameter OR doesn't have a default value)
       return isEmpty(argv[key]) && isEmpty(item.default);
